torch/csrc/jit/passes/onnx.cpp

```diff
@@ -481,6 +481,9 @@ void NodeToONNX(
               op->name());
           cloneNode(op);
         }
+      } else if (operator_export_type == ::torch::onnx::OperatorExportTypes::ONNX_ATEN_FALLBACK) {
```

Comment:
Should this branch be merged with the one above so inlining is wrapped inside a try-catch block?

Reply:
Looking at the requirements, the goal was to inline autograd functions regardless of if they have aten functions or not, which would not be possible if we use a try catch block.

Reply:
I'm confused, how are these related? Without try-catch, when exceptions happen it just throws and breaks export instead of falling back to original prim::pythonOp. How would adding try catch affect inlining aten ops?